### ISSUE TITLE
Error: EXDEV: cross-device link not permitted, rename

### ISSUE THREAD

**Issue by visibilityspots** [DeanCording/node-red-contrib-fs-ops#2]
using a pi3 archlinux with node v6.5.0 and npm 3.10.6. Imported the https://gist.github.com/DeanCording/888b4cd95250197eb429b2f40d188185 flow but when trying to upload a firmware I get this error:

```
Error: EXDEV: cross-device link not permitted, rename '/tmp/356ab0a70922b0dd58f2c5558ebb6882' -> '/storage/wemos/firmware/firmware.bin'
```

/tmp and /storage are different partitions which is I think is the root cause of the issue as I described on the gist you create for the ota esp update (https://gist.github.com/DeanCording/888b4cd95250197eb429b2f40d188185#gistcomment-1867613)

**Comment by DeanCording** [comment]
Ahh, didn't consider that case.  I'll fix it shortly.

**Comment by DeanCording** [comment]
Ahh, didn't consider that case.  I'll fix it shortly.

**Comment by DeanCording** [comment]
Fixed in 1.2.2
